deployment/clowdapp.yaml

```diff
@@ -1115,6 +1166,22 @@ objects:
   kind: Secret
   metadata:
     name: cloudwatch
+
+# Floorist
+# https://docs.engineering.redhat.com/display/COMPL/Floorist+onboarding
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: floorplan
+  data:
+    floorplan.yaml: |
+      - prefix: insights/cloudigrade/total_user_count
+        query: >-
+          select count(1) as total_user_count from auth_user;
+      - prefix: insights/cloudigrade/total_user_count_again
+        query: >-
+          select count(1) as total_user_count from auth_user;
```

Comment:
just curious, why do we have 2 of the same query ?

Reply:
To quickly demonstrate how to specify more than one query.